internal/utils/errorutils.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"errors"
+	"log"
+)
+
+const Error404Message = "По заданным значениям данных не найдено."
+const CommonErrorMessage = "Что-то пошло не так. Попробуйте снова."
+
```

Comment:
В го обычно не константы делают, а сразу переменные с ошибками нужного вида.

 ```suggestion
 var (
 	ErrNotFound = errors.New("По заданным значениям данных не найдено")
 	ErrUnknown = errors.New("Что-то пошло не так. Попробуйте снова")
 )
```

Потом при обработке ошибок можно проверять конкретные ошибки, если нужно.

Reply:
исправил